lib/modules/datasource/crate/index.spec.ts

```diff
@@ -248,6 +257,40 @@ describe('modules/datasource/crate/index', () => {
       expect(res).toBeDefined();
     });
 
+    it('processes real data for sparse registry: libc', async () => {
+      mockSparseConfig();
+
+      httpMock
+        .scope(sparseBaseUrl)
+        .get('/li/bc/libc')
+        .reply(200, Fixtures.get('libc'));
+      const res = await getPkgReleases({
+        datasource,
+        depName: 'libc',
+        registryUrls: ['sparse+https://index.crates.io'],
+      });
+      expect(res).toMatchSnapshot();
```

Comment:
no snapshots, read our dev best practices guild inside docs folder of this repo.